### ISSUE TITLE
Cohens and fleiss kappas could be useless

### ISSUE THREAD

**Issue by Suor** [idrdex/star-django#9]
If all the annotations is the same for series samples and sample validations match then both Cohens and Fleiss kappas are NaN. Could not be used for prioritising and awkward to use in annotation selecting for analysis.

**Comment by idrdex** [comment]
Yeah.  I noticed that too.  However, not so awkward for metaanalysis as both cases and controls need to be annotated.  Maybe set it as an arbitrary high number outside kappa's range (like >= 2 or something impossible) so the filters will still work.
